examples/constraints.rs

```diff
@@ -108,16 +108,18 @@ impl App {
 
     fn handle_events(&mut self) -> Result<()> {
         if let Event::Key(key) = event::read()? {
-            use KeyCode::*;
-            match key.code {
-                Char('q') | Esc => self.quit(),
-                Char('l') | Right => self.next(),
-                Char('h') | Left => self.previous(),
-                Char('j') | Down => self.down(),
-                Char('k') | Up => self.up(),
-                Char('g') | Home => self.top(),
-                Char('G') | End => self.bottom(),
-                _ => (),
+            if key.kind == KeyEventKind::Press {
```

Comment:
Generally try to use guard clauses over adding extra indentation to a block. E.g.:

```rust
if key.kind != KeyEventKind::Press {
    return Ok();
}
```